config/samples/lca_v1alpha1_imagebasedupgrade.yaml

```diff
@@ -3,19 +3,16 @@ kind: ImageBasedUpgrade
 metadata:
   name: upgrade
 spec:
-  stage: Idle
-  seedImageRef:
-    version: 4.16.0
-    image: quay.io/xyz
-  additionalImages:
-    name: ""
-    namespace: ""
   autoRollbackOnFailure: {}
   extraManifests:
   - name: sno-extramanifests
     namespace: openshift-lifecycle-agent
   oadpContent:
     - name: oadp-cm-sno-backup
       namespace: openshift-adp
-
-
+  seedImageRef:
+    version: 4.16.0
+    image: quay.io/seed-image-pull-spec:latest
```

Comment:
This image ref must be left as quay.io/xyz. Otherwise, it causes an issue with the formal build, which includes an image ref scanner

Reply:
nice catch